public/app/components/codeEditor/codeEditor.js

```diff
@@ -43,14 +60,13 @@ angular.module('kibibitCodeEditor')
         });
       });
     };
+
     // save the content of the editor on-change
     vm.aceChanged = function(_editor) {
       vm.aceDocumentValue = vm.aceSession.getDocument().getValue();
     };
 
     vm.editorOptions = {
```

Comment:
nit: hmmm... now that we have editor settings, this name might be confusing. maybe we can change this to `vm.attachedEditorFunctions`. Now, this only contains those so it might be more clear.

what do you think? I might be over thinking this :wink: